src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.PlatformServices.Tests/Implementation/AppBaseTests.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using Altinn.App.Services.Implementation;
+using Xunit;
+
+namespace Altinn.App.PlatformServices.Tests.Implementation
+{
+    public class AppBaseTests
+    {
+        [Fact]
+        public void InstanceId_Valid_ShouldParseCorrect()
+        {
+            var expectedInstanceOwnerPartyId = 123456;
+            var expectedInstanceGuid = Guid.NewGuid();
+            var instanceId = $"{expectedInstanceOwnerPartyId}/{expectedInstanceGuid}";
+
+            var (actualPartyId, actualInstaceGuid) = AppBase.DeconstructInstanceId(instanceId);
```

Comment:
this tests functionality not included in this PR, given that it is exposed through appBase.. Is the logic duplicated in appbase and instancehelper?

Reply:
This test was only written to cover the fact that you can already add custom data values, so it's basically adding coverage to already existing functionality. As part of proving implementing the test I also added the InstanceHelper class to make it easier to extract the party id and instance guid from the url.